backend/router/email_crud.py

```diff
@@ -2,12 +2,14 @@
 
 import os
 from db import get_db
-from db.orm import User, Setting
+from db.orm import User, Setting, PasswordReset
 from fastapi import APIRouter, Depends, HTTPException
 from pydantic import BaseModel, EmailStr
 from sqlalchemy.orm import Session
 from sqlalchemy.dialects.mysql import insert
 from utils.mail import send_email, get_setting
+import bcrypt
```

Comment:
pozri si sebastiánov PR #27, nepoužíva sa tam bcrypt ale passlib (ktorý obsahuje bcrypt), uprav kód aby to používalo passlib, pretože nebudeme mať zbytočne dva rovnaké moduly